changelog/changed-dap-identifier-use-interface_desc-when-product_string-is-empty.md

```diff
@@ -0,0 +1 @@
+changed-dap-identifier-use-interface_desc-when-product_string-is-empty
```

Comment:
Please write a full sentence here, this is part of the changelog and should be human readable.